test/e2e/node/pod_hostnamefqdn.go

```diff
@@ -63,15 +63,15 @@ func testPod() *v1.Pod {
 	return pod
 }
 
-var _ = SIGDescribe("Hostname of Pod [Feature:SetHostnameAsFQDN]", func() {
+var _ = SIGDescribe("Hostname of Pod [NodeAlphaFeature:SetHostnameAsFQDN]", func() {
 	f := framework.NewDefaultFramework("hostfqdn")
 
 	/*
 	   Release: v1.19
 	   Testname: Create Pod without fully qualified domain name (FQDN)
 	   Description: A Pod that does not define the subdomain field in it spec, does not have FQDN.
 	*/
-	ginkgo.It("a pod without subdomain field does not have FQDN [Feature:SetHostnameAsFQDN]", func() {
+	ginkgo.It("a pod without subdomain field does not have FQDN [NodeAlphaFeature:SetHostnameAsFQDN]", func() {
```

Comment:
actually I don't think you need to duplicate the NodeAlphaFeature:SetHostnameAsFQDN] on every test. It would be inherited from the describe call.

Reply:
great, I updated it. Thanks.